conf/layer.conf

```diff
@@ -5,14 +5,14 @@ BBPATH .= ":${LAYERDIR}"
 BBFILES += "${LAYERDIR}/recipes-*/*/*.bb \
             ${LAYERDIR}/recipes-*/*/*.bbappend"
 
-BBFILE_COLLECTIONS += "meta-esec-iotdm"
-BBFILE_PATTERN_meta-esec-iotdm = "^${LAYERDIR}/"
-BBFILE_PRIORITY_meta-esec-iotdm = "23"
+BBFILE_COLLECTIONS += "meta-connagtive"
+BBFILE_PATTERN_meta-connagtive = "^${LAYERDIR}/"
+BBFILE_PRIORITY_meta-connagtive = "23"
 
-LAYERDEPENDS_meta-esec-iotdm = "phytec openembedded-layer networking-layer meta-python rauc"
-LAYERSERIES_COMPAT_meta-esec-iotdm = "zeus"
+LAYERDEPENDS_meta-connagtive = "phytec openembedded-layer networking-layer meta-python rauc"
+LAYERSERIES_COMPAT_meta-connagtive = "zeus"
 
-DISTRO_VERSION_append = "-ESECIOTDM-2021-02-08"
+DISTRO_VERSION_append = "-connagtive-2021-02-08"
```

Comment:
I think the "c" should be in upper case for the distro version: `Connagtive`

Reply:
changed